src/script/script.cpp

```diff
@@ -275,18 +276,16 @@ bool CScript::HasValidOps() const
     return true;
 }
 
-bool GetScriptOp(CScriptBase::const_iterator& pc, CScriptBase::const_iterator end, opcodetype& opcodeRet, std::vector<unsigned char>* pvchRet)
+bool GetScriptOp(Span<const unsigned char>& script, opcodetype& opcodeRet, Span<const unsigned char>* data_out)
 {
     opcodeRet = OP_INVALIDOPCODE;
-    if (pvchRet)
-        pvchRet->clear();
-    if (pc >= end)
-        return false;
+    if (data_out)
+        *data_out = script.subspan(0, 0);
```

Comment:
nit: Is there any reason why an empty subspan on `script` ist preferred over just the default constructor (i.e. `*data_out = Span<const unsigned char>();`? For a zero-size span the address doesn't seem to matter.

Reply:
No reason, changed it.